foo.txt

```diff
@@ -0,0 +1 @@
+sdfsdf
```

Comment:
@jecisc thanks for your careful review. It was indeed intentional to force a travis rebuild. This file is not necessary anymore so I removed it in latest version. Thanks again.